spec/src/main/asciidoc/ch02-entities.adoc

```diff
@@ -1130,12 +1132,16 @@ values. Embedded objects belong strictly to their owning entity, and are
 not sharable across persistent entities. Attempting to share an embedded
 object across entities has undefined semantics.
 
-Embeddable classes must adhere to the
-requirements specified in <<a18>> for entities with the exception that embeddable classes
-are not annotated as _Entity_. Embeddable classes must be annotated as
+Embeddable classes must be annotated as
 _Embeddable_ or denoted in the XML descriptor as such. The access type
 for an embedded object is determined as described in <<a113>>.
 
+An embeddable class may be a regular Java class which adheres to the
+requirements specified in <<a18>> for entities, with the exception that
```

Comment:
not sure where a18 is but does it specify that it needs to be `Serializable`?  Entities are not required to be `Serializable`.